### ISSUE TITLE
transfer ownership ?

### ISSUE THREAD

**Issue by xuwei-k** [typelevel/scalaz-contrib#27]
Last commit is one year ago, and latest version depends on scalaz 7.0.x.

I propose
- move organization `typelevel/scalaz-contrib` to `scalaz/scalaz-contrib`
- change groupId `org.typelevel` to `org.scalaz`

what do you think? @larsrh

ref #26

**Comment by larsrh** [comment]
Yes, that's fine with me. If you have admin access in the `scalaz` organization, I can make you temporary admin on this repository so that you can transfer it.

**Comment by muradm** [comment]
What is the fate of this? I find that Validation is in scalaz._ but Checker, converter._, validator._ are missing.
